music_assistant/server/providers/deezer/manifest.json

```diff
@@ -5,6 +5,6 @@
   "description": "Support for the Deezer streaming provider in Music Assistant.",
   "codeowners": ["@Un10ck3d",  "@micha91"],
   "documentation": "https://github.com/orgs/music-assistant/discussions/1245",
-  "requirements": ["deezer-python==6.0.0", "pycryptodome==3.18.0"],
+  "requirements": ["git+https://github.com/music-assistant/deezer-python-async", "pycryptodome==3.18.0"],
```

Comment:
at some point maybe publish it to pypi ?

Reply:
Yeah good idea. But i should probably talk to the owner of deezer-python first